packages/utils/src/rundown-utils/rundownUtils.ts

```diff
@@ -1,7 +1,100 @@
-import { OntimeEvent, OntimeRundown } from 'ontime-types';
+import { OntimeEvent, OntimeRundown, OntimeRundownEntry, SupportedEvent } from 'ontime-types';
 
 import { dayInMs } from '../timeConstants.js';
 
+/**
+ * Gets first event in rundown, if it exists
+ * @param {OntimeRundownEntry[]} rundown
+ * @return {OntimeRundownEntry | null}
+ */
+export function getFirst(rundown: OntimeRundownEntry[]) {
+  return rundown.length ? rundown[0] : null;
+}
+
+/**
+ * Gets first scheduled event in rundown, if it exists
+ * @param {OntimeRundownEntry[]} rundown
+ * @return {OntimeEvent | null}
+ */
+export function getFirstEvent(rundown: OntimeRundownEntry[]) {
+  for (let i = 0; i < rundown.length; i++) {
+    if (rundown[i].type === SupportedEvent.Event) {
+      return rundown[i] as OntimeEvent;
+    }
+  }
```

Comment:
Why not use `.find`?

```ts
return rundown.find((event) => event.type === SupportedEvent.Event)
```

Reply:
Resolving due to https://github.com/cpvalente/ontime/pull/473#discussion_r1296330210